### PR TITLE
Add aad manifest doc

### PR DESCRIPTION
**Pull request by SLdragon**
Add AAD manifest feature doc following this structure:
- What is AAD Manifest and why we use it.
- Explain how this manifest will be used in TeamsFx lifecycle commands (What happens at F5, Provision, Deploy)
- Explain Placeholder.
-  Explain Authoring Supports.
- How to customize (Setting Permissions, Pre-auth Clients, Re-direct URLs)
- Any Limitations, User Need-to-know information?

Preview this file here:  https://github.com/SLdragon/wikipage/blob/add-aad-manifest/AAD-Manifest-Introduction.md

Add change log for migration with unified config and AAD manifest
- This doc is used for GA when unified config and AAD manifest migration merge together

Preview this file here: https://github.com/SLdragon/wikipage/blob/add-aad-manifest/Upgrade-to-support-unify-config-and-aad-manifest.md

### CODE REVIEW

File: Upgrade-to-support-unify-config-and-aad-manifest.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+* `localSettings.json` will be updated to `config.local.json` to ensure consistency between local and remote environment configuration settings.
+* `manifest.local.template.json` and `manifest.remote.template.json` will be merged into a single manifest template file named `manifest.template.json` to make managing manifest template files easy.
+* Update `.fx\configs\projectSettings.json` capabilities to include TabSSO and/or BotSSO based on original project capabilities.
+* AAD manifest template file will be added to `templates\appPackage\aad.template.json`
+* The required resource access information in `permissions.json` file will be merged into `aad.template.json` file and `permissions.json` file will be deprecated.
+
+Your existing `localSettings.json`, `manifest.local.template.json`, `manifest.remote.template.json` and `projectSettings.json` files will be backed up in `.backup` folder.
+
+## Know what you need to do
+Since Teams Toolkit will use `manifest.remote.template.json` as a single manifest template file after the upgrade, if you have customized the `manifest.local.template.json`, you will need to update the `manifest.template.json` to include your change.
```
Comment (MuyangAmigo) [inline Upgrade-to-support-unify-config-and-aad-manifest.md:23]: Seems not aligned with the latest `unify-config-wiki`, can you help double check: https://github.com/OfficeDev/TeamsFx/wiki/Upgrade-to-support-unify-config

File: AAD-Manifest-Introduction.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+If you want to reference `short` value, you can use this placeholder argument in the AAD manifest: `{{config.manifest.appName.short}}`
+
+### Referencing config file values in AAD manifest template
+
+Some times you may not want to hardcode the values in AAD manifest template. For example, when the value is a secret. AAD manifest template file supports referencing the values from environment variables. You can use syntax `{{env.YOUR_ENV_VARIABLE_NAME}}` in parameter values to tell the tooling that the value needs to be resolved from current environment variable.
+
+## AAD manifest authoring supports
+
+AAD manifest template file has codelens to help you better reviewing and editing it.
+
```
Comment (MuyangAmigo) [inline AAD-Manifest-Introduction.md:93]: Can we add some screenshots here?

File: AAD-Manifest-Introduction.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+### Preview codelens
+
+At the beginning of the AAD manifest template file, there is a preview codelens. Click this codelens, it will generate AAD manifest based on the environment you selected.
+
```
Comment (MuyangAmigo) [inline AAD-Manifest-Introduction.md:97]: Can we add some screenshots here?

File: Upgrade-to-support-unify-config-and-aad-manifest.md (hunk 1 above)
Comment (SLdragon) [reply Upgrade-to-support-unify-config-and-aad-manifest.md:23]: Updated

File: AAD-Manifest-Introduction.md (hunk 2 above)
Comment (SLdragon) [reply AAD-Manifest-Introduction.md:93]: Updated

File: AAD-Manifest-Introduction.md (hunk 3 above)
Comment (SLdragon) [reply AAD-Manifest-Introduction.md:99]: Updated

**Review by MuyangAmigo — APPROVED** [review]

**Pull request merged**